src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double GROWTH_MULTIPLIER = 1.5;
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] elementData;
+    private int size = 0;
```

Comment:
It's 0 by default, so you don't need to specify it.
```suggestion
    private int size;
```